errors/errors.go

```diff
@@ -28,6 +28,10 @@ var (
 	// be used (ie. persisted).
 	InvalidModelErr = Register(5, "invalid model")
 
+	// ConflictErr is returned when an operation cannot be completed due to
```

Comment:
Huh?
I don't know what that really means and saw no usage of it below

Reply:
You're right. I think I have removed the code that was using it before committing it. :wastebasket: